src/components/loader-bar/loader-bar.stories.mdx

```diff
@@ -28,37 +27,14 @@ Import `LoaderBar` into the project.
 ```javascript
 import LoaderBar from "carbon-react/lib/components/loader-bar";
 ```
-
 ## Examples
 
 ## Default bar
 
 This example of the LoaderBar component demonstrates how it will appear as default.
 
 <Canvas>
-  <Story name="default">
-    <LoaderBar mt={2} />
-  </Story>
-</Canvas>
-
-### Small bar
-
-This is an example of a small LoaderBar component.
-
-<Canvas>
-  <Story name="small">
-    <LoaderBar size="small" mt={2} />
-  </Story>
-</Canvas>
-
-### Large bar
-
-This is an example of a large LoaderBar component.
-
-<Canvas>
-  <Story name="large">
-    <LoaderBar size="large" mt={2} />
-  </Story>
```

Comment:
Not sure why these stories have been removed? We would want to move the `small` and `large` stories to `stories.tsx` as well and import them back in here like you have done with `default`.